lib/lunchbot_web/controllers/admin/export_orders_controller.ex

```diff
@@ -0,0 +1,65 @@
+defmodule LunchbotWeb.Admin.ExportOrdersController do
+  use LunchbotWeb, :controller
+
+  alias Lunchbot.LunchbotData
+
+  def index(conn, _params) do
+    render(conn, "index.html")
+  end
+
+  def export(conn, _params) do
+    order_struct = LunchbotData.list_preloaded_orders()
+
+    all_orders =
+      for order <- order_struct do
+        timestamp = order.inserted_at
+        user = order.user.name
+
+        order_items =
+          for order_item <- order.order_items do
+            item_with_options =
+              for order_item_option <- order_item.order_item_options do
```

Comment:
there are a lot of nested for loops here, maybe we can push some of this logic into a SQL query?